pkg/ibmvpc/parser.go

```diff
@@ -614,12 +634,32 @@ func getNACLconfig(rc *datamodel.ResourcesContainerModel,
 }
 
 func getTgwObjects(c *datamodel.ResourcesContainerModel,
-	res map[string]*vpcmodel.VPCConfig) map[string]*TransitGateway {
+	res map[string]*vpcmodel.VPCConfig, resourceGroup string) map[string]*TransitGateway {
 	tgwMap := map[string]*TransitGateway{} // collect all tgw resources
+
+	tgwIDToRG := map[string]string{}
```

Comment:
How about having a `map[string]bool` that keeps a set of TGWs to skip?

Reply:
need this map anyway.. added the other map